modules/godot/databags/databag_timer.cpp

```diff
@@ -0,0 +1,66 @@
+#include "databag_timer.h"
+
+void TimersDatabag::_bind_methods() {
```

Comment:
Can you please bind the methods, so the scripted systems can also use this new databag?